josh-mongo/index.js

```diff
@@ -61,36 +60,42 @@ class JoshProvider {
       throw new Error('Keys should be strings or numbers.');
     }
     this.db.update({
-      _id: key
+      _id: key,
     }, {
       _id: key,
-      value: val
+      value: val,
     }, {
-      upsert: true
+      upsert: true,
     });
+    return this;
   }
 
   get(key) {
     console.log(`Retrieving ${key}'s data`);
     return this.db.findOne({
-      _id: key
+      _id: key,
     });
   }
 
+  inc(key, path) {
+    this.set(key, path, this.get(key, path) + 1);
+    return this;
+  }
```

Comment:
```suggestion
  inc(key, path) {
    return this.set(key, path, this.get(key, path) + 1);
  }
```
`#set` already returns `this`, the next line was quite unnecessary.

Reply:
Quite right, it's also in josh-SQLite